vscode/src/codeActions.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ILanguageService } from "qsharp-lang";
+import * as vscode from "vscode";
+import { ICodeAction } from "../../npm/qsharp/lib/web/qsc_wasm";
```

Comment:
Don't import via a relative path into the npm package. Import from 'qsharp-lang'.

Reply:
Fixed, thanks